lib/rules/require-ember-lifeline.js

```diff
@@ -30,6 +30,9 @@ function mergeDisallowedCalls(objects) {
 }
 
 function isLintableFile(context) {
+  if (!context || !context.eslint) {
```

Comment:
Do you know what changes to cause the method to be missing? 

I’m concerned that there may be more fundamental changes that we need to deal with (rather than adding a guard like this)...

Reply:
Good catch. I saw some other projects solve it like this, but according to https://eslint.org/docs/developer-guide/working-with-rules it's available on the context object. Ran the current tests (based on eslint 3), everything works fine.